tests/test_happy_path.py

```diff
@@ -0,0 +1,176 @@
+from brownie import Wei, chain, RewardsManager
+from utils.config import (
+    gift_index,
+    one_inch_token_owner,
+    farming_rewards_owner,
+    initial_rewards_duration_sec,
+    rewards_amount,
+)
+
+# Test Case:
+# 1. Deploy and configure rewards manager
+# 2. Reward farming contract sets reward manager as distributor for LDO token gift
+# 3. DAO transfers LDO to reward manager contract
+# 4. 1INCH distributor transfers tokens to farming contract directly
+# 5. Someone starts new reward period (1INCH via their distributor, LDO via reward manager)
+# 6. Users A and B are obtaining stETH and DAI tokens somehow
+# 7. Users A and B deposit their stETH and DAI tokens to liquidity pool to receive LP tokens
+# 8. Users A and B are staking their LP tokens to reward farming contract
+# 9. User A exists from reward farming contract after half of rewarding period
+# 10. User A withdraws tokens from liquidity pool
+# 11. User B exists from reward farming contract after full rewarding period
+# 12. User B withdraws tokens from liquidity pool
+# 13. User B has more rewards than A
+def test_happy_path(
```

Comment:
The happy path is a bit out of scope. It tests the FarmingRewards contract more than RewardsManager. I think the test case might be cut to the next scenario: 
1. deploy rewards manager
2. set it as rewardsDistributor for FarmingRewards
3. transfer LDO via Aragon's voting to RewardManager
4. start the next reward period
5. test that LDO balances of RewardsManager and FarmingRewards were changed correctly
6.  wait till the end of the reward period
7. start a new one and test balances again.
I don't mind keeping the current test case, but it lacks a test for the second reward period

Reply:
Addressed in https://github.com/maddevsio/lido/pull/3/commits/fb256d420eebc03dbb6aa022078f12bf5bb40d38

Reply:
All looks good